engines/bops_submissions/config/initializers/rswag_ui.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Rswag::Ui.configure do |c|
+  # c.openapi_endpoint "/submissions/api-docs/v1/swagger_doc.yaml", "Submission API V1 Docs"
+  c.openapi_endpoint "/api/docs/v2/swagger_doc.yaml", "Submission API V2 Docs"
```

Comment:
You have to remember that the RSwag configuration is a singleton so if they're both loaded then they'll conflict